scanner/config/config.go

```diff
@@ -99,6 +99,12 @@ type MatcherConfig struct {
 	Database Database `yaml:"database"`
 	// Enable if false disables the Matcher service and vulnerability updater.
 	Enable bool `yaml:"enable"`
+	// IndexerAddr forces the matcher to retrieve index reports from a remote indexer
+	// instance at the specified address, instead of the local indexer when it is
+	// enabled.
+	IndexerAddr string `yaml:"indexer_addr"`
+	// RemoteIndexer internal and generated flag, true when the remote indexer is enabled.
+	RemoteIndexer bool
```

Comment:
since it's internal, you can probably do `remoteIndexer`. Also, in this context, `RemoteIndexer` makes me think it holds the configuration for the remote indexer. I think in this context maybe `enableRemoteIndexer` makes more sense

Reply:
I see this is used, actually. That being said, I still think `EnableRemoteIndexer` or `RemoteIndexerEnabled` would be clearer